WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/PencilEditButton.swift

```diff
@@ -3,11 +3,28 @@ import SwiftUI
 /// Renders a button with the pencil system name image
 ///
 struct PencilEditButton: View {
+    @ScaledMetric private var scale: CGFloat = 1.0
+    private let size: CGFloat
+
     let action: () -> Void
 
+    init(size: CGFloat = Layout.editIconImageSize, action: @escaping () -> Void) {
+        self.size = size
+        self.action = action
+    }
+
     var body: some View {
         Button(action: action) {
-            Image(systemName: "pencil")
+            Image(uiImage: .pencilImage)
+                .resizable()
+                .frame(width: size * scale,
+                       height: size * scale)
         }
     }
```

Comment:
Perhaps this should have a default `accessibilityLabel` of `Edit` – it works fine everywhere now, but for future it would make it less likely that we miss the VoiceOver compliance.